docs/usage.rst

```diff
@@ -0,0 +1,97 @@
+.. _usage:
+
+=====
+Usage
+=====
+
+Here's normal usage of the server:
+
+.. code-block:: python
+
+   from waitress import serve
+   serve(wsgiapp, listen='*:8080')
+
+This will run waitress on port 8080 on all available IP addresses, both IPv4
+and IPv6.
+
+
+.. code-block:: python
+
+   from waitress import serve
+   serve(wsgiapp, host='0.0.0.0', port=8080)
+
+This will run waitress on port 8080 on all available IPv4 addresses.
+
+If you want to serve your application on all IP addresses, on port 8080, you
+can omit the ``host`` and ``port`` arguments and just call ``serve`` with the
+WSGI app as a single argument:
+
+.. code-block:: python
+
+   from waitress import serve
+   serve(wsgiapp)
+
+Press Ctrl-C (or Ctrl-Break on Windows) to exit the server.
+
+The default is to bind to any IPv4 address on port 8080:
```

Comment:
This line can be moved up, and then we can remove this extra code block since it's the exact same example.

Reply:
I made several other revisions while you were reviewing. Please check commit 88248b8, especially whether I got the bit about IPv4 or IPv6 addresses correct.